src/dmd/dmodule.d

```diff
@@ -59,9 +59,8 @@ version(Windows) {
  * Returns:
  *      NULL if it's not different from filename.
  */
-private const(char)* lookForSourceFile(const(char)** path, const(char)* filename)
+private const(char)* lookForSourceFile(const(char)* filename)
```

Comment:
You removed a parameter, but didn't update the documentation.

Reply:
nice catch...fixed